cmlutils/project_entrypoint.py

```diff
@@ -321,6 +325,8 @@ def project_import_cmd(project_name):
         end_time = time.time()
         import_file = log_filedir + constants.IMPORT_METRIC_FILE
         write_json_file(file_path=import_file, json_data=import_data)
+        if target_owner and target_owner != username:
```

Comment:
Please add logs  indicating transfer of ownership to make it more verbose.

Reply:
Log is present inside function - transfer_ownership